dataprofiler/tests/profilers/test_base_column_options.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import BaseColumnOptions
+from dataprofiler.tests.profilers.test_boolean_option import TestBooleanOption
+
+
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.'
+            'DataLabelerColumn.update', return_value=None)
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.DataLabeler')
+class TestBaseColumnOptions(TestBooleanOption):
+    
+    @classmethod
+    def setUpClass(cls):
+        cls.option_class = BaseColumnOptions
+
+    @classmethod
+    def get_options(self, *args, **params):
+        if self.option_class == None:    
+            raise ValueError("option_class class variable cannot be set to 'None'")
+        options = self.option_class()
+        options.set(params)
+        return options
+            
+    def test_init(self, *mocks):
+        super().test_init(*mocks)
+    
+    def test_set_helper(self, *mocks):
+        super().test_set_helper(*mocks)
+
+    def test_set(self, *mocks):
+        super().test_set(*mocks)
+    
+    def test_validate_helper(self, *mocks):
+        super().test_validate_helper(*mocks)
+    
+    def test_validate(self, *mocks):
+        super().test_validate(*mocks)
```

Comment:
This is redundant (I am overriding the super method to call the super method). It does make the file easier to read though as it's clear what tests are being run (which is good if these tests need to be modified in the future).